cli/src/commands/deploy.ts

```diff
@@ -46,11 +45,9 @@ export default class Deploy extends Command {
       }
 
       const filesToExclude = new Set(['node_modules', 'backend', 'src'].map((f) => pathResolve(projectDir, f)));
-      const dotShiftPath = pathResolve(projectDir, '.shift');
       await copy(projectDir, stagingDir, {
         filter(src) {
-          // TODO: we should just put all the .shift files in a single .shift directory
-          return !filesToExclude.has(src) && !src.startsWith(dotShiftPath);
+          return !filesToExclude.has(src) && !path.basename(src).startsWith('.');
```

Comment:
won't this also skip user files starting with dot under `build` as well?
maybe skip only dotfiles in the topmost directory? (for .git and shift related stuff)

Reply:
Thanks!

There are really quite subtle behaviors that need to keep track of when translating from bash.